scripts/tests/test_load_csv_prereg.py

```diff
@@ -0,0 +1,83 @@
+# -*- coding: utf-8 -*-
+import os
+import json
+
+from nose.tools import *  # noqa
+
+from tests.base import OsfTestCase
+
+from scripts.prereg import load_csv_prereg
+
+__location__ = os.path.realpath(os.path.join(os.getcwd(), os.path.dirname(__file__)))
```

Comment:
This can be written more concisely, such as the following:
`os.path.dirname(os.path.abspath(__file__))`

([see docs](https://docs.python.org/2/library/os.path.html#os.path.abspath))